packages/vertica-nodejs/test/integration/client/api-tests.js

```diff
@@ -9,19 +9,24 @@ suite.test('null and undefined are both inserted as NULL', function (done) {
   pool.connect(
     assert.calls(function (err, client, release) {
       assert(!err)
-      client.query('CREATE TEMP TABLE my_nulls(a varchar(1), b varchar(1), c integer, d integer, e date, f date)')
-      client.query('INSERT INTO my_nulls(a,b,c,d,e,f) VALUES ($1,$2,$3,$4,$5,$6)', [
+      client.query('DROP TABLE IF EXISTS my_nulls; CREATE TEMP TABLE my_nulls(a varchar(1), b varchar(1), c integer, d integer, e date, f date)')
```

Comment:
Didn't we decide to just use local temp tables?

Reply:
Yeah, added this before that, will change